app/widgets/news/Controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "cx/ui";
+
+const getArticles = () =>
+	fetch("https://newsapi.org/v1/articles?source=the-next-web&sortBy=latest&apiKey=c03cf67d0e414ff3a578badda1b368cf")
+		.then(x => {
+			if (!x.ok) throw new Error("Failed to fetch BTC price from https://newsapi.org.");
```

Comment:
Error message should be `Failed to fetch news...`

Reply:
That's what happens when you edit someone else's code :/